lib/travis.js

```diff
@@ -5,7 +5,9 @@ var TRAVIS_BRANCH_CONFIGS = {
     ORIGIN: 'https://teach.mofostaging.net',
     SHOW_DEV_RIBBON: 'on',
     MAILINGLIST_URL: 'https://sendto.mozilla.org/page/s/maker-party-signup-for-teach-site-staging-testing',
-    MAILINGLIST_PRIVACY_NAME: 'custom-3508'
+    MAILINGLIST_PRIVACY_NAME: 'custom-3508',
+    // PLEDGE_MAILINGLIST_URL: '',
+    // PLEDGE_MAILINGLIST_PRIVACY_NAME: ''
```

Comment:
I will plug in the right values and uncomment these lines once I know which BSD form to use